packages/project-input-validator/package.json

```diff
@@ -26,7 +26,7 @@
         "unlink": "pnpm unlink --global"
     },
     "dependencies": {
-        "i18next": "20.6.1",
+        "i18next": "23.5.1",
```

Comment:
This is not aligned with other modules. Is the newer version needed?
Is yes then please create a TBI to update other modules to align with this one.

Reply:
Yes it is needed, why are we using such an old version?

Reply:
My guess is that we just used the same version as internally at the time, and then did move to a newer version since it wasn't required. Thanks for the issue so that we can address it later.